scripts/get_previous_timestamp.py

```diff
@@ -47,8 +47,7 @@ def get_workflow_runs(token: str, repo: str, workflow: str):
             f.write(f"status code: {r.status_code}")
         return None
     run_info = json.loads(r.text)
-    print(run_info)
-    runs = [run for run in run_info if run["name"] == workflow]
+    runs = [run for run in run_info["workflow_runs"] if run["name"] == workflow]
```

Comment:
```suggestion
    print(run_info)
    runs = [run for run in run_info["workflow_runs"] if run["name"] == workflow]
```

Reply:
print statement too large to be meaningful as seen here https://github.com/ewlu/gcc-precommit-ci/actions/runs/7280980870/job/19840548856#step:6:50